shared/cloud/ess-getting-started-obs.asciidoc

```diff
@@ -1,23 +1,18 @@
 // Include this file in your docs:
 // include::{docs-root}/shared/cloud/ess-getting-started-obs.asciidoc[]
 
-// To include APM Server instructions, add this attribute:
-// :include-apm-instructions:
-
 . {ess-trial}[Get a free trial].
 
 . Log into {ess-console}[{ecloud}].
 
 . Click *Create deployment*.
 
-. Select *Elastic Observability* and give your deployment a name.
+. Give your deployment a name.
 
-ifdef::include-apm-instructions[]
-. Click *Create deployment* and copy the password for the `elastic` user.
+. Click *Create deployment*
 
-. Select *APM* from the menu on the left and make note of the APM endpoint and APM Server secret token.
-endif::include-apm-instructions[]
+. Save your deployment credentials
 
-ifndef::include-apm-instructions[]
-. Click *Create deployment* and copy the password for the `elastic` user and Cloud ID information.
-endif::include-apm-instructions[]
+You can find your Cloud ID and your APM endpoint in your deployments
+list at {ess-deployments}[cloud.elastic.co], and reset the provided
+password on the *Security* page for your deployment.
```

Comment:
```suggestion
You can find your Cloud ID and APM endpoint in your deployments
list at {ess-deployments}[cloud.elastic.co]. To reset the provided
password, go to the *Security* page for your deployment.
```

Reply:
Let's break this into two sentences to avoid putting a comma between the subject and verb that appears in the second independent clause.